VimeoNetworkingExample-iOS/VimeoNetworkingCommonTests/VIMCategory+Tests.swift

```diff
@@ -0,0 +1,142 @@
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import XCTest
+@testable import VimeoNetworking
+
+class VIMCategory_Tests: XCTestCase
+{
+    private var parsedCategory: VIMCategory?
+    private var categoryJSONDict: VimeoClient.ResponseDictionary?
+    
+    override func setUp()
+    {
+        super.setUp()
+        
+        guard let categoryJSONDict = ResponseUtilities.loadResponse(from: "categories-animation-response.json") else
+        {
+            self.parsedCategory = nil
+            self.categoryJSONDict = nil
+            return
+        }
+        
+        self.categoryJSONDict = categoryJSONDict
+        self.parsedCategory = try! VIMObjectMapper.mapObject(responseDictionary: categoryJSONDict) as VIMCategory
+    }
+    
+    func test_VIMCategory_ValidateParsing()
+    {
+        XCTAssertNotNil(self.parsedCategory)
+        
+        XCTAssertEqual(self.parsedCategory?.name, self.categoryJSONDict?["name"] as? String)
+        XCTAssertEqual(self.parsedCategory?.uri, self.categoryJSONDict?["uri"] as? String)
+        XCTAssertEqual(self.parsedCategory?.link, self.categoryJSONDict?["link"] as? String)
+        XCTAssertTrue(self.parsedCategory?.isTopLevel ?? false)
+
+        //There is no modifiedTime field in the response data so this should be nil
+        XCTAssertNil(self.parsedCategory?.modifiedTime)
+        
+        XCTAssertNotNil(self.parsedCategory?.pictureCollection)
+        XCTAssertGreaterThan(self.parsedCategory?.pictureCollection?.pictures?.count ?? 0, 0)
+        
+        XCTAssertNotNil(self.parsedCategory?.subcategories)
+        XCTAssertGreaterThan(self.parsedCategory?.subcategories?.count ?? 0, 0)
+    }
+    
+    func test_VIMCategory_IsFollowingReturnsTrue()
+    {
+        XCTAssertNotNil(self.parsedCategory)
+        XCTAssertTrue(self.parsedCategory?.isFollowing() ?? false)
+    }
+    
+    func test_VIMCategory_ParsedSubcategories()
+    {
+        XCTAssertNotNil(self.parsedCategory?.subcategories)
+        
+        let subCatsArray = self.categoryJSONDict?["subcategories"] as? [Dictionary<String, String>]
```

Comment:
Small potatoes, but we probably don't need to include type information in the name of the constant. How about just `subcategories`, similar to the property on `VIMCategory`.